src/envoy/http/authn/http_filter.cc

```diff
@@ -104,5 +97,44 @@ void AuthenticationFilter::setDecoderFilterCallbacks(
   decoder_callbacks_ = &callbacks;
 }
 
+void AuthenticationFilter::continueDecoding() {
+  if (state_ != Istio::AuthN::State::PROCESSING) {
+    ENVOY_LOG(error, "State {} is not PROCESSING.", state_);
+    return;
+  }
+  state_ = Istio::AuthN::State::COMPLETE;
+  if (stopped_) {
+    decoder_callbacks_->continueDecoding();
```

Comment:
stopped_ = false ???

Reply:
Stopped is set (to true) when FilterHeadersStatus::StopIteration was returned. Only when so, continue decoding need to be called. Same logic as jwt filter.